tests/e2e/tests/pilot/testdata/networking/v1alpha3/wildcard-https-multihosts-no-vs.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: googlebing
+spec:
+  hosts:
+  - www.google.com
+  - "*.bing.com"
+  ports:
```

Comment:
the wikipedia test is already testing this in some way.. so are other bookinfo tests I think.

Reply:
yes except it doesn't test multiple hosts and resolution NONE